packages/wrangler/src/__tests__/versions/secrets/bulk.test.ts

```diff
@@ -75,6 +75,18 @@ describe("versions secret put", () => {
 		expect(std.err).toMatchInlineSnapshot(`""`);
 	});
 
+	test("no wrangler configuration warnings shown", async () => {
+		await writeFile("secrets.json", JSON.stringify({ SECRET_1: "secret-1" }));
+		await writeFile("wrangler.json", JSON.stringify({ invalid_field: true }));
```

Comment:
Nit, but could this use the writeWranglerConfig helper?

Reply:
Not really because that requires that the input object matches the `RawConfig` type, which then requires an awkward cast.